aspnet/client-side/bundling-and-minification.rst

```diff
@@ -1,8 +1,315 @@
-.. include:: /../common/stub-topic.txt
+Bundling and Minification
+=========================
+
+By `Erik Reitan`_ 
+
+Bundling and minification are two techniques you can use in ASP.NET 5 to improve performance. Bundling makes it easy to combine or bundle multiple files into a single file. Minification performs a variety of different code optimizations to scripts and CSS, which results in smaller file sizes.
```

Comment:
Do you want to spell out what's exactly meant by "performance" here? (TTFB, page load time?)